application/frontend/config/main.php

```diff
@@ -20,6 +20,7 @@
         ],
         // http://www.yiiframework.com/doc-2.0/guide-runtime-requests.html
         'request' => [
+            'enableCookieValidation' => false,
```

Comment:
I'm curious... why the cookie validation change?

Reply:
Yii2 was updated from 2.0.13 to 2.0.15. I don't know the background of why this was added, other than 2.0.15 has some security fixes.

Reply:
It looks like this change may have been the result of partially-copied code from another repo. IIRC, once Yii started requiring `enableCookieValidation` to be set, we added a way for us to provide a cookie validation key via an env. var.

For example, see the various occurrences of "cookieValidation" (case-insensitive) in this commit on idp-id-sync:
https://github.com/silinternational/idp-id-sync/blob/3732472b7b3b2bac43444134b3cc953b95ad7ac2/application/frontend/config/main.php

I'd recommend going ahead and adding...
```php
$cookieValidationKey = Env::get('COOKIE_VALIDATION_KEY');
```
... up above this array in this file, then changing this line per the following (as part of dealing with the upgrade to Yii 2.0.15):
```suggestion
            'cookieValidationKey' => $cookieValidationKey,
            'enableCookieValidation' => !empty($cookieValidationKey),
```